src/js/tabs/failsafe.js

```diff
@@ -18,6 +18,11 @@ TABS.failsafe.initialize = function (callback, scrollPosition) {
     }
     
     function load_rxfail_config() {
+        MSP.send_message(MSPCodes.MSP_GPS_RESCUE, false, false, 
```

Comment:
Now you've mixed up the function names and the MSP commands that they execute. But the way you are conditionally calling either or is spot on! :-)

Reply:
Ahhhhh incredible fail :-o